wikimedia/wikipedia-ios
# Add Apple fine print to native donate form

**Phabricator:** 
https://phabricator.wikimedia.org/T345846

### Notes
Added "Apple is not in charge of raising money for this purpose." fine print to the bottom of the native donate form. Marking as hold until design review is complete.

### Test Steps
1. Log into iOS Sandbox Tester Apple ID (see 1Password) in test device iOS Settings. Do not enable 2 factor auth.
2. In Wallet app, add [Test Apple Pay card](https://developer.apple.com/apple-pay/sandbox-testing/).
3. Change device region to NL or IT.
4. On Staging scheme, fresh install and launch app.
5. On Explore feed, background, then foreground, then pull to refresh on Explore feed to fetch latest campaigns.
In Settings, you should now see a "Donate to Wikipedia" cell, tap it. Tap "Donate with Apple Pay" option in alert view.
6. Note new fine print on donate form.

### Screenshots

![IMG_0240](https://github.com/wikimedia/wikipedia-ios/assets/3620196/e0e344e5-d29c-440d-841b-3b801655b89d)
